src/App.tsx

```diff
@@ -1,11 +1,22 @@
+import { BrowserRouter } from "react-router-dom"
+import { CardImage } from "./components/CardImage"
+import { CardColor } from "./components/CardColor"
 import { Footer } from "./components/Footer"
-import { Router } from "./routing/Router"
+import { routes } from "./routing/routes"
 
 export const App = () => {
+  const { department, field } = routes
   return (
     <div>
-      <Router />
-      <Footer />
+      <BrowserRouter>
```

Comment:
Co to za BrowserRouter? Tam został zrobiony router, jak w niego wejdziesz to masz tam obiekt `router` i jeden obiekt który. Zmieniasz go na:
```
{
    path: routes.root(),
    element: <div>[TUTAJ WSTAW SWÓJ KOMPONENT]</div>
  }
```
Z czego ja to zmienię jeszcze jak dodam główny Layout, więc na ten moment nie dodawaj tego footera i headera. Daj mi znać jak to zmergujesz.